tests/certification/bindings/azure/cosmosdb/cosmosdb_test.go

```diff
@@ -0,0 +1,82 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package keyvault_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/dapr/components-contrib/bindings"
+	cosmosdbbinding "github.com/dapr/components-contrib/bindings/azure/cosmosdb"
+	"github.com/dapr/components-contrib/secretstores"
+	secretstore_env "github.com/dapr/components-contrib/secretstores/local/env"
+	bindings_loader "github.com/dapr/dapr/pkg/components/bindings"
+	secretstores_loader "github.com/dapr/dapr/pkg/components/secretstores"
+	"github.com/dapr/dapr/pkg/runtime"
+	dapr_testing "github.com/dapr/dapr/pkg/testing"
+	"github.com/dapr/kit/logger"
+
+	"github.com/dapr/components-contrib/tests/certification/embedded"
+	"github.com/dapr/components-contrib/tests/certification/flow"
+	"github.com/dapr/components-contrib/tests/certification/flow/sidecar"
+)
+
+const (
+	sidecarName = "cosmosdb-sidecar"
+)
+
+func TestKeyVault(t *testing.T) {
```

Comment:
To clarify, these are really only for making sure an init works, right? Or am I missing something here?

Reply:
Right now they are only for making sure the init works with the different authentication methods.

I will be adding more tests, but didn't make this PR too massive.